Dockerfile

```diff
@@ -14,14 +14,14 @@ RUN pnpm exec turbo build --filter=@apps/$APP_NAME^...
 RUN pnpm --filter=@apps/$APP_NAME exec vite build
 RUN pnpm --filter=@apps/$APP_NAME deploy --prod out
 
-FROM gcr.io/distroless/nodejs20-debian12 AS deplpyer
+FROM gcr.io/distroless/nodejs20-debian12 AS deployer
 
 ENV NODE_ENV=production
 ENV ORIGIN=http://localhost:8080
 
 WORKDIR /app
 
-COPY --from=builder /app/out/ .
+COPY --chown=nonroot:nonroot --from=builder /app/out/ .
```

Comment:
change owner of copied files to the nonroot user defined in the gcr.io/distroless image.